### PR TITLE
fix(analytics): Add platform only if available in issue banner

### PR DESCRIPTION
**Pull request by denamwangi**
Not all projects have platform set - sending as part of the data only if available

fixes RELOAD-D

### CODE REVIEW

**Review by evanpurkhiser — DISMISSED** [review]

File: src/sentry/static/sentry/app/components/events/eventEntries.jsx [hunk 1]
```diff
@@ -83,10 +83,10 @@ class EventEntries extends React.Component {
 
     analytics('issue_error_banner.viewed', {
       org_id: parseInt(orgId, 10),
-      platform: project.platform,
       group: event.groupID,
       error_type: errorTypes,
       error_message: errorMessages,
+      ...(platform ? {platform: project.platform}),
```
[the remark this answers is not in the document]
Comment (denamwangi) [reply src/sentry/static/sentry/app/components/events/eventEntries.jsx]: We set this when a project is created based on the platform they choose in that initial flow so new projects should have it but old ones won't :/

[the remark this answers is not in the document]
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/events/eventEntries.jsx]: I mean the variable `platform`. I don't see it declared?

File: src/sentry/static/sentry/app/components/events/eventEntries.jsx [hunk 2]
```diff
@@ -83,10 +83,10 @@ class EventEntries extends React.Component {
 
     analytics('issue_error_banner.viewed', {
       org_id: parseInt(orgId, 10),
-      platform: project.platform,
       group: event.groupID,
       error_type: errorTypes,
       error_message: errorMessages,
+      ...(platform && {platform: project.platform}),
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/eventEntries.jsx]: I think evan means `platform` is `undefined`

```suggestion
      ...(project.platform && {platform: project.platform}),
```

Comment (denamwangi) [reply src/sentry/static/sentry/app/components/events/eventEntries.jsx]: ouf- thanks

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/events/eventEntries.jsx]: I'd be okay with null traversal operator here too ;)

`platform: project?.platform`

Comment (denamwangi) [reply src/sentry/static/sentry/app/components/events/eventEntries.jsx]: Just a bit more context on the reasoning here: `reload` does a check on the type and sends a sentry warning if there is a mismatch (this particular one has a ton).

**Review by markstory — APPROVED** [review]

**Pull request merged**